### PR TITLE
Security and maintainability fixes

### PR DESCRIPTION
**Pull request by gregtyler**
# Purpose

Fix a few issues highlighted by Trivy and SonarCloud

#patch

## Approach

Use Alpine 3.19 for fixtures container: this contains the libexpat patch and others.

Move the properties file to the name used for cloud analysis (I'd previously used the one for CI analysis).

Some minor maintainability fixes that were easy to fix.

## Learning

The various ways you can configure SonarCloud caused me some confusion, but [it seems we want](https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-parameters/#setting-configuration-in-a-file) `.sonarcloud.properties` as our configuration file name for it to be picked up properly.

### CODE REVIEW

**Review by acsauk — DISMISSED** [review]
:shipit:

File: fixtures/Dockerfile [hunk 1]
```diff
@@ -6,14 +6,12 @@ COPY fixtures/package.json package.json
 COPY fixtures/package-lock.json package-lock.json
 RUN npm ci --ignore-scripts
 
-FROM python:3-alpine3.17
+FROM python:3-alpine3.19
```
Comment (mattmachell) [inline fixtures/Dockerfile:9]: You might want a specific python version here to avoid unexpected upgrades and breakage.

Comment (gregtyler) [reply fixtures/Dockerfile]: Yeah, fair, I've pinned to 3.13

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**